coa-helm/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: terra-batch-libchart
   repository: file://../terra-batch-libchart
-  version: 0.0.1
-digest: sha256:ed27fc69bde2ca0d336ce281c2a437c65f7067503181f4a285634ab3273b7fab
-generated: "2022-07-27T07:47:12.343214-04:00"
+  version: 0.0.2
+digest: sha256:d115e87c168da0ef62a22c0d12b5c9b6d227bb3fa4a5d7f9ce9b7d5dc2510661
+generated: "2022-08-19T15:51:23.864572-04:00"
```

Comment:
TOL: I wonder whether we could have a github action to auto-run `helm --dependency-update` every time a PR merges which updates the libchart...? 🤔

Reply:
this is a good thought... I have been unclear on best practices in terms of whether or not to check-in `Chart.lock` and the `charts` directory into version control